recipes/python-libxdo/LICENSE

```diff
@@ -0,0 +1,12 @@
+Copyright (c) <YEAR>, <OWNER>
```

Comment:
I did see this upstream but it is a blank template  :-/

Reply:
The original author uploaded the template to the source repository https://github.com/rshk/python-libxdo/blob/master/LICENSE . We could replace the placeholders with his name and the year when he uploaded the license to the source repository, would that be sufficient?

Reply:
We should not do that. I'm rather open an issue upstream and add a link to the issue in the meta.yaml file. That way we have some provenance that we warned them and we are using what they provided.

Reply:
@ocefpaf I added an issue upstream https://github.com/rshk/python-libxdo/issues/33 as well as a comment in the `meta.yaml` file.